### ISSUE TITLE
Batch effect removal with only one cell type

### ISSUE THREAD

**Issue by ryeking2010** [theislab/scgen#60]
Hello,

Is it possible to remove batch effects when giving all the cells the same label?

Will this prevent the loss function from properly optimizing the parameter values in the neural network?

We have isolated the same cell type from many batches and hope to remove the batch effect to study subtypes.

Thank you in advance for your assistance!

**Comment by M0hammadL** [comment]
Hi Sorry for late reply, 

I think it would not be ideal if you don't know the subtype labels, I would suggest checking out scArches : 
https://github.com/theislab/scarches
